shared/cluster/src/main/java/io/pravega/common/cluster/zkImpl/ClusterZKImpl.java

```diff
@@ -109,6 +114,10 @@ public void deregisterHost(Host host) {
         close(node);
     }
 
+    public boolean isZKConnected() {
+        return isZKConnected.get();
```

Comment:
Please add tests to cover this new code in all classes.

Reply:
Added  tests